alcs-frontend/src/app/features/home/assigned/assigned.component.ts

```diff
@@ -101,7 +101,7 @@ export class AssignedComponent implements OnInit {
     this.sortedFiles = sorted;
   }
 
-  private mapCovenant(c: CovenantDto): AssignedToMeFile {
+  private mapCovenant(c: NoticeOfIntentDto): AssignedToMeFile {
```

Comment:
is this correct that mapCovenant accepts NoticeOfIntentDto?

Reply:
Fixed